src/_vmprof.c

```diff
@@ -148,10 +148,11 @@ static PyObject *enable_vmprof(PyObject* self, PyObject *args)
 {
     int fd;
     int memory = 0;
+    int lines = 0;
     double interval;
     char *p_error;
 
-    if (!PyArg_ParseTuple(args, "id|i", &fd, &interval, &memory))
+    if (!PyArg_ParseTuple(args, "id|ii", &fd, &interval, &memory, &lines))
```

Comment:
Yes, sounds good
On 12 Jul 2016 1:41 PM, "Dmitry Trofimov" <notifications@github.com> wrote:

> In src/_vmprof.c
> <https://github.com/vmprof/vmprof-python/pull/87#discussion_r70422230>:
>
> >      double interval;
> >      char *p_error;
> >
> > -    if (!PyArg_ParseTuple(args, "id|i", &fd, &interval, &memory))
> > +    if (!PyArg_ParseTuple(args, "id|ii", &fd, &interval, &memory, &lines))
>
> The parameters are named in the vmprof.enable method, so there is no urge
> to make them named here. If it is possible I would avoid making this more
> complex.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/vmprof/vmprof-python/pull/87/files/a50f56916c3941288726bfbda8e604092182b96d#r70422230>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAFPuHihH4DbZuoZRthriHyZ7LrsTBcyks5qU31fgaJpZM4JJfnp>
> .
>